its/src/test/java/com/sonar/it/csharp/Tests.java

```diff
@@ -97,18 +99,20 @@ public static void deleteLocalCache() {
     TestUtils.deleteLocalCache();
   }
 
-  static BuildResult analyzeProject(TemporaryFolder temp, String projectName, @Nullable String profileKey, String... keyValues) throws IOException {
-    Path projectDir = Tests.projectDir(temp, projectName);
+  static BuildResult analyzeProject(String projectDir, TemporaryFolder temp, String projectName, @Nullable String profileKey, String... keyValues) throws IOException {
+    Path projectFullPath = Tests.projectDir(temp, projectDir);
 
-    ScannerForMSBuild beginStep = TestUtils.createBeginStep(projectName, projectDir)
+    ScannerForMSBuild beginStep = TestUtils.createBeginStep(projectName, projectFullPath)
       .setProfile(profileKey)
       .setProperties(keyValues);
 
     ORCHESTRATOR.executeBuild(beginStep);
+    TestUtils.runMSBuild(ORCHESTRATOR, projectFullPath, "/t:Restore,Rebuild");
+    return ORCHESTRATOR.executeBuild(TestUtils.createEndStep(projectFullPath));
+  }
 
-    TestUtils.runMSBuild(ORCHESTRATOR, projectDir, "/t:Restore,Rebuild");
-
-    return ORCHESTRATOR.executeBuild(TestUtils.createEndStep(projectDir));
+  static BuildResult analyzeProject(TemporaryFolder temp, String projectName, @Nullable String profileKey, String... keyValues) throws IOException {
+    return analyzeProject(projectName, temp, projectName, profileKey, keyValues);
```

Comment:
This is the old overload, it calls the new one with the same `projectName` at two argument positions.